basic-stashing/README.md

```diff
@@ -0,0 +1,42 @@
+# Git Kata: Basic stashing 
+
+## Setup:
+In your terminal, run
+```
+$ . ./setup.sh

```

Comment:
generally we use either source `. setup.sh` which moves peoples workdir into the exercise repo, which is useful, or `./setup.sh` which leaves people out of the exercise dir. Which causes some confusion especially among non-terminal native users :)